src/services/email-line-item-delivery.js

```diff
@@ -143,13 +156,13 @@ module.exports = {
    * @param {string} params.placementId
    * @param {Date} params.date
    */
-  async getLineItemFor({ placementId, date: d } = {}) {
+  async getLineItemsFor({ placementId, date: d } = {}) {
     if (!placementId) throw createError(400, 'No placement ID was provided.');
     if (!d) throw createError(400, 'No date was provided.');
     const { $d: date } = dayjs(d).tz('UTC');
 
     // find the _first_ matching line item. if more than one, pick the oldest (i.e. sort by _id).
```

Comment:
This comment is no longer accurate, update to reflect the new behavior. Also pass limit through and use it in the query opts.